example_app/build_web.sh

```diff
@@ -1,75 +0,0 @@
-#!/bin/bash
-set -eu
-script_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
-cd "$script_path/.."
-
-OPEN=false
-FAST=false
-
-while test $# -gt 0; do
-  case "$1" in
-    -h|--help)
-      echo "build_web.sh [--fast] [--open]"
-      echo "  --fast: skip optimization step"
-      echo "  --open: open the result in a browser"
-      exit 0
-      ;;
-    --fast)
-      shift
-      FAST=true
-      ;;
-    --open)
-      shift
-      OPEN=true
-      ;;
-    *)
-      break
-      ;;
-  esac
-done
-
-# ./setup_web.sh # <- call this first!
-
-CRATE_NAME="example_app"
-CRATE_NAME_SNAKE_CASE="${CRATE_NAME//-/_}" # for those who name crates with-kebab-case
-
-# This is required to enable the web_sys clipboard API which egui_web uses
-# https://rustwasm.github.io/wasm-bindgen/api/web_sys/struct.Clipboard.html
-# https://rustwasm.github.io/docs/wasm-bindgen/web-sys/unstable-apis.html
-export RUSTFLAGS=--cfg=web_sys_unstable_apis
```

Comment:
Have you set this up in trunk?

Reply:
I added a .cargo/config.toml now

Is that okay ?